### DISCUSSION TITLE
How to obtain detailed information on NREL 5MW wind turbine tower bolts, flange thickness, etc.?

### DISCUSSION THREAD

**Discussion by tatesnow** [OpenFAST/openfast#995] [Q&A] [answered]
I would like to do a study on the fatigue damage of tower bolts, how can I get the information on tower bolts and flange thickness for NREL 5MW wind turbines?

**Comment by jjonkman** [comment, marked as answer]
Dear @tatesnow,

These details have not been specified for the NREL 5-MW baseline wind turbine, which is fully documented in its specification report here: https://www.nrel.gov/docs/fy09osti/38060.pdf.  You'll have to make your own assumptions about tower bolts and flanges for your own purposes.

Best regards,
